imports/lib/models/DiscordRoleGrants.ts

```diff
@@ -1,6 +1,8 @@
-import type { DiscordRoleGrantType } from '../schemas/DiscordRoleGrant';
-import Base from './Base';
+import DiscordRoleGrant from '../schemas/DiscordRoleGrant';
+import type { ModelType } from './Model';
+import Model from './Model';
 
-const DiscordRoleGrants = new Base<DiscordRoleGrantType>('discord_role_grants');
+const DiscordRoleGrants = new Model('jr_discord_role_grants', DiscordRoleGrant);
```

Comment:
This was previously a child of `Base`, which I think means the direct migration should be to a `SoftDeletedModel` here, rather than just `Model`?

Reply:
Yep, good catch. Fixed